main_logic/tts_client.py

```
@@ -1202,6 +1202,260 @@ async def async_worker():
+                # 打断时不发 end（避免等待推理完成），直接关闭连接
+                if sid != current_speech_id and sid is not None:
+                    if _ws_is_open(ws):
+                        await close_session(ws, receive_task, send_end=False)
+                        ws = None
+                        receive_task = None
+                    current_speech_id = sid
+                    for _retry in range(3):
+                        try:
+                            await create_connection()
+                            break
+                        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)